src/gasPriceOracle/oracle.e2e.ts

```diff
@@ -12,7 +12,7 @@ const dummyLogger = winston.createLogger({
 
 type FeeData = providers.FeeData;
 
-class MockedProvider extends providers.JsonRpcProvider {
+class MockedProvider extends providers.StaticJsonRpcProvider {
```

Comment:
Will the `chainId`/`network` ever change? IIRC the `StaticJsonRpcProvider` doesn't expect any changing networks

Reply:
Nope, there's a 1:1 relationship between a Provider object and a remote RPC provider, and they never change in the bots. Note however that this is in the e2e test file, so it's not even part of the production code. I switched to StaticJsonRpcProvider to reduce the chance of rate-limiting (requesting chainId can be very spammy), because sometimes these tests are run against public RPC endpoints.